build.conf

```diff
@@ -29,6 +29,8 @@ SKYWIRE_VERSION=v0.5.0
 
 SKYWIRE_ARM64_DOWNLOAD_URL="https://github.com/skycoin/skywire/releases/download/$SKYWIRE_VERSION/skywire-$SKYWIRE_VERSION-linux-arm64.tar.gz"
 SKYWIRE_ARM_DOWNLOAD_URL="https://github.com/skycoin/skywire/releases/download/$SKYWIRE_VERSION/skywire-$SKYWIRE_VERSION-linux-arm.tar.gz"
+
+DMSGHTTP_CONFIG_URL="https://raw.githubusercontent.com/skycoin/skywire/master/dmsghttp-config.json"
```

Comment:
This will fail for the time being, since we dont have that on master yet. Please change it to develop and make a ticket to swap it out once we make the new release.

Reply:
Done. Its issue is #133.